en/compatibility.md

```diff
@@ -52,3 +52,64 @@ h1, h2, h3, h4, h5, h6 { text-align: center; }
 {% endfor %}
 
 </table>
+
+## Segwit Addresses
+
+<table class="compatibility">
+  <tr>
+    <th></th>
+    <th colspan="3">Receiving support</th>
+    <th colspan="4">Sending support</th>
+  </tr>
+  <tr>
+    <th>Tool</th>
+    <th>P2SH-wrapped</th>
+    <th>bech32</th>
+    <th>default address</th>
+    <th>P2WPKH</th>
+    <th>P2WSH</th>
+    <th>bech32 change</th>
+    <th>segwit v1</th>
+  </tr>
+
+{% assign tools = site.data.compatibility | sort %}
+{% for wrapped_tool in tools %}
+  {% assign tool = wrapped_tool[1] %}
+  <tr>
+    <td><a href="{{tool.internal_url}}#segwit-addresses">{{tool.name}}</a></td>
+    {% if tool.segwit %}
+
+    {% assign segwit_receive_default = "" %}
```

Comment:
Indent everything between here and line 108 (and line 110) to show they're in the if-else blocks.